### DISCUSSION TITLE
Introduction (Outreach internship 2024)

### DISCUSSION THREAD

**Discussion by SisiVero** [camicroscope/caMicroscope#682] [General]
Hi everyone, Veronica here. I'm sorry if this is not the right place to do this but I thought it okay to create a discussion thread where we can introduce ourselves and get to know each other. I look forward to contributing to this great project. Thank you

**Comment by Khadijahhhh** [comment]
Hello, I am Khadijah, I got accepted into the Outreachy internship program. I look forward to contributing to this project.Thank you.

**Reply by Ebere330** [reply]
Hello everyone, I am Ebere, an Outreachy intern, I look forward to contributing to this project

**Reply by SisiVero** [reply]
Hello Khadijah and Ebere, it's good to have you both. Welcome.

**Reply by maha-sachin** [reply]
Hi all, I am Maha here. Looking forward to contribute to this standalone project.

**Reply by SisiVero** [reply]
Welcome Maha. It's good to have you

**Reply by NgweBecky** [reply]
hi everyone, I am Becky an outreachy intern.

**Reply by SisiVero** [reply]
HI Becky, it's good to have you.

**Reply by gbengaoluwadahunsi** [reply]
It's nice having you all here.

**Reply by DreamingMoon** [reply]
Hello everyone, I'm Semilore. I look forward to contributing to this project

**Reply by We13b-MD** [reply]
Its nice having everybody here much love to everyone

**Reply by PASEDA-MODUPE** [reply]
Hello everyone I'm Modupe happy to be here

**Comment by Kingjhenny** [comment]
Hello everyone, I am Jennifer, Nice to meet you all and I'm exited to be contributing to this project

**Reply by gbengaoluwadahunsi** [reply]
Nice to have you here Jennifer.

**Comment by sanketmp** [comment]
> Hi everyone, Veronica here. I'm sorry if this is not the right place to do this but I thought it okay to create a discussion thread where we can introduce ourselves and get to know each other. I look forward to contributing to this great project. Thank you

Hi everyone, I'm Sanket and I am an Outreachy applicant for May 2024. I am very excited and looking forward to contributing to caMicroscope.

**Reply by ESE-MONDAY** [reply]
Hello Sanket

**Reply by gbengaoluwadahunsi** [reply]
Hey Veronica, sure it will be a good time with caMicroscope.

**Reply by SisiVero** [reply]
Hello Sanket, you're very welcome.

**Comment by ESE-MONDAY** [comment]
Hi everyone, my name is Ese,

I got into the Outreachy contribution phase and i'm looking forward to contributing to caMicroscope

**Reply by gbengaoluwadahunsi** [reply]
Hello  Ese, nice having you here as co-contributors.

**Reply by SisiVero** [reply]
Good day Ese. You're very welcome

**Comment by gbengaoluwadahunsi** [comment]
Hello, I am Gbenga , glad that  I will be contributing to caMicroscope

**Reply by SisiVero** [reply]
Hello Gbenga, good to have you. I read the question you asked previously. How are you finding the cloning and all that? Smooth?

**Reply by gbengaoluwadahunsi** [reply]
Hello @SisiVero , 
Thanks,  nice so far, the files are large to it takes some time to clone to one's local machine.

**Reply by We13b-MD** [reply]
Hi I am Idundun Michael a full stack developer (mern) i will be contributing to caMicroscope i will love to help and i wish everyone the best

**Reply by SisiVero** [reply]
Welcome Idundun.
@gbengaoluwadahunsi yeah.
It's really large

**Comment by AleruDivine** [comment]
Hello everyone! My name is Aleru Divine, I'm an Outreachy applicant and I look forward to connecting with you all and making meaningful contributions to caMicroscope.

**Reply by gbengaoluwadahunsi** [reply]
Hello @AleruDivine, welcome to the open-source contribution with caMicroscope, together we are adding our marks to the future of better medial imaging.

**Reply by SisiVero** [reply]
Good to have you, Aleru

**Comment by ranjeetxsingh** [comment]
Hello folks! I am an Outreachy applicant for may 2024 cohort, I am looking forwards to contributing to this project.

**Reply by SisiVero** [reply]
We are happy to have you here, Ranje

**Comment by jumokeadeoyo** [comment]
Hi everyone, I'm Adeoyo Olajumoke, an Outreachy applicant and I look forward to making my contributions to help in the advancement of bio-medical imaging.

**Reply by SisiVero** [reply]
Hello Olajumoke, you're most welcome

**Reply by Khadijahhhh** [reply]
Hi Olajumoke, you are welcome

**Comment by Ebere330** [comment]
Hi everyone, I'm Ebere Adekogbe, an outreachy applicant , I look forward to making positive contributions.

**Reply by SisiVero** [reply]
Hi Ebere, it's good to have you

**Reply by Kingjhenny** [reply]
Welcome Ebere

**Comment by diamondgem8** [comment]
hello everyone, am Diamond outreachy contributor, happy to be here

**Reply by SisiVero** [reply]
Hi Diamond, it's great to have you

**Reply by Kingjhenny** [reply]
Hello diamond, it's nice to meet you 😊

**Comment by SanjanaSogimatt** [comment]
Hey everyone, I am Sanjana and this is my first time here as Outreachy contributor.

**Reply by SisiVero** [reply]
Hi Sanja, you're welcome

**Reply by Kingjhenny** [reply]
Hi sanjana.. 🤗

**Comment by medinasheriff** [comment]
hello everyone, my name is Medina, a 2024 outreachy applicant. I look forward to contributing to this project, and working with you all.

**Reply by Kingjhenny** [reply]
Hello Medina.. welcome

**Reply by SisiVero** [reply]
Hi Medina, you're most welcome.

**Reply by medinasheriff** [reply]
Thank you

**Comment by darrenodi** [comment]
Greetings eeryone. I am Darren from Cameroon and I am very excited about this project. I hope to learn from both my mentors and colleagues and make a wonderful impact.

**Reply by Kingjhenny** [reply]
Welcome onboard Darren

**Reply by SisiVero** [reply]
Hi @darrenodi ,it's good to have you

**Reply by darrenodi** [reply]
Thank you @Kingjhenny @SisiVero .

**Comment by aanya963** [comment]
Hi everyone, I'm Aanya from India, I am an Outreachy applicant. I am very excited and looking forward to contributing to caMicroscope.

**Reply by Kingjhenny** [reply]
Hi Aanya, welcome. Nice to meet you

**Reply by medinasheriff** [reply]
Hello Aanya.

**Reply by SisiVero** [reply]
Hello @Aanya9693 , we are happy to have you

**Comment by Kb-Jr** [comment]
Hello Everyone, My name is Kabir. Really looking forward to contributing to this project and collaborating with everyone.

**Reply by SisiVero** [reply]
Hello @Kb-Jr , we are looking forward to collaborating with you too. You're very welcome

**Reply by Kb-Jr** [reply]
Thank you very much @SisiVero

**Comment by RWUBAKWANAYO** [comment]
Hi everyone, I'm Olivier, an outreachy applicant. I'm happy to be here and excited to share and learn from all of you.

**Reply by Kb-Jr** [reply]
Welcome Oliver

**Reply by SisiVero** [reply]
Hi @RWUBAKWANAYO , you're most welcome.

**Comment by gbengaoluwadahunsi** [comment]
Hello Olivier, nice  to have you here

**Comment by codinBabe** [comment]


**Reply by SisiVero** [reply]
Hello @codinBabe , you're welcome.

**Reply by codinBabe** [reply]
Thank you @SisiVero

**Comment by Topaztemmy** [comment]
Hello everyone, my name is Temitope. An outreachy applicant, looking forward to learn and contribute to this project

**Reply by SisiVero** [reply]
Hello @Topaztemmy , it's good to have you

**Comment by kemsguy7** [comment]
Hello Everyone, I am Matthew Idungafa,  an outreachy applicant and I look forward to contributing to this organization.

**Reply by SisiVero** [reply]
Hello @kemsguy7 , it's good to have you

**Reply by gbengaoluwadahunsi** [reply]
Welcome here @kemsguy7

**Comment by akriti-dhurandher** [comment]
Hello, I am Akriti and this is my first time contributing. I am excited to be contributing to this project

**Reply by SisiVero** [reply]
Hi @akriti-dhurandher , you're very welcome.

**Comment by Lahitan** [comment]
Hi Everyone I am Tawakalitu, an Outreachy applicant, I am happy to contribute to this project

**Reply by SisiVero** [reply]
Hi @Lahitan , it's good to have you

**Reply by Lahitan** [reply]
Thank you @SisiVero

**Comment by DIANAAJAMBO** [comment]
Hello, I am Diana Ajambo, an outreachy applicant from Uganda. I look forward to collaborating with everyone and learning from each other.

**Reply by kemsguy7** [reply]
Welcome @DIANAAJAMBO , it's nice to have you here

**Reply by DIANAAJAMBO** [reply]
Thank you Matthew @kemsguy7

**Comment by Moliki-Salman** [comment]
Hello everyone, I am Damilola from Nigeria. I am excited to contribute to the project and also learn from the community

**Reply by gbengaoluwadahunsi** [reply]
Hello @Moliki-Salman  you are welcome here

**Comment by JUNNY12** [comment]
Hello everyone, I am Juwon Akingbade from Nigeria. It's Nice meeting you all.

**Reply by gbengaoluwadahunsi** [reply]
Hello @JUNNY12, its nice having you here.

**Reply by JUNNY12** [reply]
Thank you @gbengaoluwadahunsi. I am looking forward to learning and working with you

**Comment by mhdamaan79** [comment]
Hello everyone! I'm Amaan from India, an Outreachy applicant for May 2024. Excited to dive into caMicroscope's contribution journey!

**Reply by gbengaoluwadahunsi** [reply]
hello@Aman,lets go
you are welcome here

**Reply by mhdamaan79** [reply]
Thanks @gbengaoluwadahunsi, Excited to learn and grow together on this journey. Let's go!

**Comment by WanjaNjunge** [comment]
Hello everyone, I am Wanja, from Kenya. Looking forward to working and engaging with you all on this project

**Reply by medinasheriff** [reply]
Hi, Wanja. Welcome.

**Reply by gbengaoluwadahunsi** [reply]
Welcome here @WanjaNjunge

**Comment by goodylove** [comment]
hi everyone ,please has anyone started contributing already?

**Comment by gbengaoluwadahunsi** [comment]
There are lot of repos here and there where you could find your voice, some
are code heavy and others are suggestions and proposals,

You might not start with code related contributions, contributions to docs
and the apps UI will also be nice.

**Comment by Lochipi** [comment]
Hello everyone! My name is Cornelius Emase, I'm an Outreachy applicant and I look forward to connecting with you all and making meaningful contributions to caMicroscope.

**Reply by gbengaoluwadahunsi** [reply]
Hello @Lochipi , glad to be con-contributors on this project

**Comment by nwanduka** [comment]
Hello everyone, Victoria Nduka here. I'm a UX designer and researcher. Looking to contribute meaningfully to this project.

**Comment by Moliki-Salman** [comment]
welcome @Lochipi  and @nwanduka .

**Reply by rchaurase** [reply]
Hi there,

My name is Rahul.

It's really amazing to see how much interest there is in this initiative. An exciting path lies ahead thanks to the diversity of viewpoints and the flood of receptive minds. In addition to guaranteeing a good time, interacting with such a lively group provides countless chances for development and learning. Together, let's set out on our journey to explore the vast expanse of uncharted territory.

**Comment by LeonardAzah** [comment]
Hello please I need help. while running the Distro repo I keep getting this error.
Building 0.1s (3/3) FINISHED                                                                                       
 => ERROR [distro_loader internal] load git source https://github.com/camicroscope/SlideLoader.git#v3.11.0        0.1s
 => ERROR [distro_back internal] load git source https://github.com/camicroscope/caracal.git#v3.11.0              1.8s
 => CACHED [distro_dicomsrv internal] load git source https://github.com/camicroscope/dicomsrv.git#v3.11.0        0.0s
------
 > [distro_loader internal] load git source https://github.com/camicroscope/SlideLoader.git#v3.11.0:
#1 1.753 0684457a6c83e6d45ebc73bd84b276fc3ba8eab9       refs/tags/v3.11.0
------
------
 > [distro_back internal] load git source https://github.com/camicroscope/caracal.git#v3.11.0:
------
failed to solve: rpc error: code = Unknown desc = failed to read dockerfile: open https:/github.com/camicroscope/dicomsrv.git#v3.11.0/Dockerfile: no such file or directory

**Reply by gbengaoluwadahunsi** [reply]
Hello @LeonardAzah 
try and restart the session,  do a build before running
1. Use docker-compose -f caMicroscope.yml build
2. docker-compose -f caMicroscope.yml up
3. Once everything is up, go to http://localhost:4010/ to see the landing page.

**Comment by tubapervaiz1508** [comment]
Hello Everyone, My name is Tuba Pervaiz. I am an Outreachy applicant and look forward to work with you all and give meaningful contribution to this project.

**Reply by blessdamy** [reply]
Hi @everyone I am Blessing from Nigeria😇  I am super excited to be here. I am super glad to collaborate with everyone and learn from our amazing mentor👏

**Reply by gbengaoluwadahunsi** [reply]
Gald to have you both here  @blessdamy @tubapervaiz1508

**Comment by FlourishRalph** [comment]
Hi guys, I'm Flourish, glad to be here.👋🏾

**Reply by SiziweLupondo** [reply]
Welcome @FlourishRalph  Looking forward to see you flourish through out this contribution Period.

**Comment by AteeqahYussuf** [comment]
Hi everyone, my name is Ateeqah, also an outreach applicant and I’m so excited to contribute to this project

**Reply by SiziweLupondo** [reply]
Hello. Welcome

**Comment by anshikavashistha** [comment]
I am Anshika Vashistha,an outreachy applicant.I want to work on this project.[Create Stanalone Executable for caMicroscope](https://www.outreachy.org/outreachy-june-2024-internship-round/communities/camicroscope/#create-stanalone-executable-for-camicroscope)

**Reply by SiziweLupondo** [reply]
Hello @anshikavashistha . i thought this was for Accessibility Testing and Improvement for Biomedical Imaging project or it is mixed.

**Comment by anshikavashistha** [comment]
Could anyone guide me how should I start contribution?

**Comment by pamisijohn** [comment]
> Hi everyone, Veronica here. I'm sorry if this is not the right place to do this but I thought it okay to create a discussion thread where we can introduce ourselves and get to know each other. I look forward to contributing to this great project. Thank you

Hello! I'm John, an Outreachy applicant. I hope I am not late to start contributing here!

**Reply by SiziweLupondo** [reply]
Hello John, Welcome

You are never late for making meaningful contributions.

**Reply by belrah** [reply]
hi @SiziweLupondo any idea how we can go about this project? i am equally new here
thank you

**Comment by SiziweLupondo** [comment]
Hello @nanli-emory and  @everyone, my name is Siziwe from South Africa. I am happy to be here. 
Congratulations once more to all of us who made it to the contribution phase. I am looking to contribute to "Accessibility Testing and Improvement for Biomedical Imaging" , Looking forward to collaborate , and learn from everyone here . i hope you will learn from me too

**Comment by izzdwizz** [comment]
Hi everyone, my name is Izuchukwu.

I got into the Outreachy contribution phase, I'm really happy to be here and I look forward to contributing to caMicroscope

**Reply by gbengaoluwadahunsi** [reply]
Hello @izzdwizz , you are welcome here

**Comment by Charity256** [comment]
Hello, I am Charity, a new applicant for the Outreachy internship program. I look forward to contributing to the caMicroscope project.

**Reply by gbengaoluwadahunsi** [reply]
Welcome here @Charity256

**Comment by CodebyCc** [comment]
Hello, I am George, an applicant for the Outreachy internship program. I look forward to contributing to the caMicroscope project but all issues on the project have been closed, howdo i go about it

**Reply by gbengaoluwadahunsi** [reply]
Hi @CodebyCc , you are  welcome here

**Comment by Gungun2310** [comment]
Hi, I am Gungun Singh, a new applicant for the Outreachy internship program. I look forward to contributing to the caMicroscope project and getting started with another phase of learning.

**Reply by gbengaoluwadahunsi** [reply]
Welcome @Gungun2310

**Comment by JennySimen** [comment]
Hi everyone, my name is Jennifer, I am an outreachy applicant. I am excited to be here and I look forward to contributing to the caMicroscope project.

**Reply by gbengaoluwadahunsi** [reply]
Hello @JennySimen , you are welcome here, happy contributions.

**Comment by Ayesh-07** [comment]
I'm Ayesha Nudrat. I have skills in HTML, CSS, Bootstrap, and JavaScript. I want to contribute to caMicroscope to enhance my skills and knowledge, particularly in areas related to web development.

**Comment by gbengaoluwadahunsi** [comment]
Welcome here at Ayesha,
Happy contributions

**Comment by SisiVero** [comment]
Hello @everyone , I welcome all the new comers. 
It's good to have you.
Below is the link to the step by step guide written by @gbengaoluwadahunsi on how to get started:
https://github.com/orgs/camicroscope/discussions/705#discussioncomment-8727824

**Comment by Arnav2824** [comment]
Hello everyone. My name is Arnav and I am really exctied to contribute to camicoscope for outreachy 2024. I hope my experience in MERN Stack and Python will help me in this venture.

**Reply by nwanduka** [reply]
You're welcome @Arnav2824. Glad to have you here.

**Comment by deveshidwivedi** [comment]
Hello everyone! My name is Deveshi Dwivedi and I am an Outreachy 2024 applicant. Super happy to be here!

**Comment by We13b-MD** [comment]
Hello everyone is good to have you all here we so glad to have you

**Comment by fynSheema** [comment]
Hi Everyone, my name is Winifred, looking forward to learning and contributing.

**Comment by shailpujan88** [comment]
hello everyone,I am Shail Pujan from India an outreachy applicant. . So happy to be part of this contribution.
Could you please provide some insights on:

1. Which issues are currently seeking contributions or have entry-level tasks suitable for newcomers like me?
2. Are there any specific coding guidelines or project documentation I should familiarize myself with before diving into contributing code? And any tips regarding setting up the codebase in my local repo
3. Do you recommend any resources or tutorials that could help me get up to speed with the project's technologies or workflows?

Furthermore, I would greatly appreciate any suggestions or tasks you believe would be suitable.

**Comment by RichieOrito** [comment]
Hello Everyone My name is Richard Orito an Outreachy Applicant, and I'm excited to be joining the caMicroscope discussion group ready to contribute and collaborate. Thank you.

**Reply by Lochipi** [reply]
welcome @RichieOrito. We look forward to seeing your meaningful contribution. To get started, [you can get started here to set up your development environment](https://gautammishra.medium.com/camicroscope-development-environment-2c0d9bb1b64b)

**Comment by gbengaoluwadahunsi** [comment]
Welcome here @RichieOrito
